backend/src/main/java/com/google/minicrm/data/Form.java

```diff
@@ -85,6 +85,39 @@ public Entity asEntity() {
     return formEntity;
   }
 
+  /**
```

Comment:
This is non-blocking: I forgot to mention this before. It seems you can actually use Google's generator for value classes called AutoValue. It will generate getters/setters for you automatically and also give equals, hashcode for free. If this seems feasible it could be worth using. Example: https://www.baeldung.com/introduction-to-autovalue

Reply:
Hm, seems pretty useful, will look into it more.